app/models/flight.rb

```diff
@@ -15,7 +15,7 @@ def self.get_date
 
   def self.formatted_departure_date
     date = get_date
-    date.map{|date|date.strftime("%B %-d, %Y")}.uniq
+    date.map{|format|format.strftime("%B %-d, %Y")}.uniq
```

Comment:
Space after closing `|` missing.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.